slowmo.owl

```diff
@@ -127,240 +219,669 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+        <rdfs:subClassOf rdf:resource="http://example.com/slowmo#Figure"/>
+        <rdfs:label>trend figure</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://example.com/slowmo#slowmo_0000010 -->
+
+    <owl:Class rdf:about="http://example.com/slowmo#slowmo_0000010">
+        <rdfs:subClassOf rdf:resource="http://example.com/slowmo#slowmo_0000142"/>
+        <obo:IAO_0000115>a measurable action that is engaged in by a performer</obo:IAO_0000115>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">obsolete behavior</rdfs:label>
+        <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
+    </owl:Class>
+    
+
+
+    <!-- http://example.com/slowmo#slowmo_0000024 -->
+
+    <owl:Class rdf:about="http://example.com/slowmo#slowmo_0000024">
+        <rdfs:subClassOf rdf:resource="http://example.com/slowmo#slowmo_0000142"/>
+        <obo:IAO_0000115>The social and physical environment in which an intervention takes place.</obo:IAO_0000115>
+        <rdfs:comment>Adapted from West, R. and Michie, S. (2016). A Guide to Development and Evaluation of Digital Behavior Interventions in Healthcare. Silverback Publishing.</rdfs:comment>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">obsolete setting</rdfs:label>
+        <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
+    </owl:Class>
+    
+
+
+    <!-- http://example.com/slowmo#slowmo_0000035 -->
+
+    <owl:Class rdf:about="http://example.com/slowmo#slowmo_0000035">
+        <rdfs:subClassOf rdf:resource="http://example.com/slowmo#slowmo_0000142"/>
+        <obo:IAO_0000115>An appraisal with represent the performance behavior.</obo:IAO_0000115>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">obsolete appraisal of performance behavior</rdfs:label>
+        <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
+    </owl:Class>
+    
+
+
+    <!-- http://example.com/slowmo#slowmo_0000142 -->
+
+    <owl:Class rdf:about="http://example.com/slowmo#slowmo_0000142">
         <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">obsolete legacy terms</rdfs:label>
         <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
     </owl:Class>
+    
+
+
+    <!-- http://example.com/slowmo#spek -->
+
+    <owl:Class rdf:about="http://example.com/slowmo#spek">
+        <obo:IAO_0000115>def needed</obo:IAO_0000115>
+        <rdfs:label xml:lang="en">spek</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Annotations
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    <rdf:Description rdf:about="http://purl.org/dc/elements/1.1/">
```

Comment:
The entire following section seems peculiar to me.  It looks like an artifact of protege merging in parts of DC11.  Adding definitions for DC11 classes strikes me as redundant.  These should all have already been defined in `http://purl.org/dc/elements/1.1/` and that should have been pulled in by a namespace statement.

I wonder if the `xmlns:terms="http://purl.org/dc/terms/"` results in protege grabbing the dc11 terms in this fashion.

In any event.  This is not a blocker.  Having these here is find for the time being.

Reply:
@grosscol took a look and I think these we coming from an unneeded import. Removed for now (f756512 ) and removing the `catalog-v001.xml` files from `.gitignore`.